spec/services/service_provider_updater.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe ServiceProviderUpdater do
+  describe '#ping' do
+    it 'returns true for success' do
+      expect(subject.ping).to eq true
+    end
+  end
+end
```

Comment:
This filename is missing the `_spec` at the end, which is probably why CC thinks it's not tested.